app/shared/containers/Global/Blockchain/Form.js

```diff
@@ -39,7 +39,7 @@ class GlobalBlockchainForm extends Component<Props> {
         derived = set(state, 'errors', {});
       }
       if (validate.NODE === 'FAILURE') {
-        derived = set(state, 'errors.endpoint', true);
+        derived = set(state, 'errors.endpoint', 'undefined');
```

Comment:
What was the reasoning behind this change? PR seems to work, but when the node hits the FAILURE event I'm not sure why it would be set to `'undefined'` (a string).

Reply:
If you look at line 227, the `errors.endpoint` state is used to generate a locale key. Before this change the locale key that was being generated was `tools_form_blockchain_node_error_true` when I think that it should be generating `tools_form_blockchain_node_error_undefined` since that's a key that exists on `tools.json`. If you want me to leave `errors.endpoint` as a boolean, I can do that and modify line 227 accordingly instead :)

Reply:
That makes sense - and while I think we could do better, works for now!